### PR TITLE
Update help command

### LINKED ISSUE #28: Help Command
As a first time user, I can see a help message explaining which features I should try first so that I can start by trying features that are more suited for new users.

### PR DESCRIPTION
**Pull request by Shivang211**
closed #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,6 @@
 
 ## Acknowledgements
 
-_This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org)._
- 
+_This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```
Comment (lamwj98) [inline README.md:24]: Why the removing of _? I think it is the mark down syntax you removed.

Comment (lamwj98) [reply README.md:24]: But if you delete then this markdown is essentially not usable, because there was _ infront of the text.

Comment (lamwj98) [reply README.md:24]: Previously, I think this is the same format, and it passed CI checks

File: data/expenditureexpert.json [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+      "description": "Watched Movie",
+      "expenseCategory": "Entertainment",
+      "amount": "12"
+    },
+    {
+      "description": "Took Train",
+      "expenseCategory": "Travel",
+      "amount": "2"
+    }
+  ]
```
Comment (lamwj98) [inline data/expenditureexpert.json:38]: is the additional of break line necessary between every brackets?

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
+        HELP_STRING.append("COMMANDS").append("\n").append("\n").append(AddCommand.MESSAGE_USAGE).append("\n\n");
+        HELP_STRING.append("clear: ").append("Clears the records of Expenditure Expert").append("\n\n");
+        HELP_STRING.append(DeleteCommand.MESSAGE_USAGE).append("\n\n");
+        HELP_STRING.append(EditCommand.MESSAGE_USAGE).append("\n\n");
+        HELP_STRING.append("exit: ").append("Exits the app").append("\n\n");
+        HELP_STRING.append(FindCommand.MESSAGE_USAGE).append("\n\n");
+        HELP_STRING.append("list: ").append("Lists all expenses").append("\n\n");
+        return HELP_STRING.toString();
+    }
+
```
Comment (lamwj98) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java:35]: Would it be a better OOP solution to have each message's proper format to be inside of each a command so? This increases the readability of this class and leaves the abstraction to the parser classes.

Comment (Shivang211) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java:35]: okay will update that

File: data/expenditureexpert.json (hunk 2 above)
Comment (Shivang211) [reply data/expenditureexpert.json:38]: again, i was failing ci checks

File: README.md (hunk 1 above)
Comment (Shivang211) [reply README.md:24]: Previously they were passing for me too, what do you want me to do here, I will try that

**Pull request closed without merging**